app/controllers/api/v1/locations_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Api
+  module V1
+    class LocationsController < ApplicationController
+      before_action :set_location, only: [:update, :destroy]
+
+      def create
+        @school = CodeSchool.find(params[:code_school_id])
+        @school.locations.build(location_params)
```

Comment:
Since we are not passing the instance variable to a Rails view, you can set this to be a local variable of `school`, instead of an instance variable of `@school`.

Reply:
ok